draft-ietf-webtrans-http2.md

```diff
@@ -200,12 +200,12 @@ fields indicate the desired WebTransport server. An `Origin` header {
 {!RFC6454}} MUST be provided within the request.
 
 Upon receiving an extended CONNECT request with a `:protocol` field set to
-`webtransport`, the HTTP server can check if it has a WebTransport server
-associated with the specified `:authority` and `:path` values. If it does not,
-it SHOULD reply with status code 404 (Section 6.5.4,{{!RFC7231}}). If it does,
-it MAY accept the session by replying with status code 200. The WebTransport
-server MUST verify the `Origin` header to ensure that the specified origin is
-allowed to access the server in question.
+`webtransport`, the HTTP server checks if the identified resource supports
+WebTransport sessions. If the resource does not, it SHOULD reply with status
```

Comment:
Wording nit: Just to keep the pronoun clear, 
> it SHOULD reply
Is that the server replying or the resource? Current wording implies resource
```suggestion
WebTransport sessions. If the resource does not, the server SHOULD reply with status
```